snap/snapcraft.yaml

```diff
@@ -0,0 +1,31 @@
+name: prometheus-juju-backup-all-exporter
```

Comment:
This is probably not necessary right now, but I'd suggest to utilize `install` hook to copy/create "default" config file in the `$SNAP_DATA` location as expected by config loader.

Reply:
That's a good suggestion. I will added it.